src/System.Numerics.Vectors/src/System/Numerics/VectorMath.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#if HAS_INTRINSICS
+using System.Diagnostics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace System.Numerics
+{
+    internal static class VectorMath
+    {
+        public static Vector128<float> Lerp(Vector128<float> a, Vector128<float> b, Vector128<float> t)
```

Comment:
I've seen guidance on other PRs that putting `[MethodImplAttribute(MethodImplOptions.AggressiveInlining)]` on functions that take/return Vector128 is a good idea.

See [https://github.com/briancylui/machinelearning/issues/3#issuecomment-410118363](https://github.com/briancylui/machinelearning/issues/3#issuecomment-410118363) where we were seeing perf degradation when we refactored some "helper" vector methods.

Reply:
It can help, but looking at the existing codegen it looks good/correct as is.

Reply:
(that is, the method is small enough to be automatically inlined)

Reply:
Yeah I've just checked both outputs - both Lerp and Equals\NotEquals are inlined.

Reply:
OK, thanks for checking. It's good to know that this refactoring didn't hurt perf.